config/locales/en.yml

```diff
@@ -966,6 +966,9 @@ en:
       title: Webhooks
       webhook: Webhook
   admin_mailer:
+    auto_close_registrations:
+      body: Considering no moderator has recently been active on your your server, its settings have been automatically changed to not allow new user registrations without approvals.
```

Comment:
I would add a justifications here, so people don't have to go look for docs to find out why this behaviour exists.

> Due to lack of recent moderator activity, registrations on %{instance} have been automatically switched to requiring manual review, to prevent %{instance} from being used as a platform for potential bad actors. You can switch back to open registrations any time.